syntax = "proto2";
package example2;

enum ContactType {
    Home = 0;
    Cop = 1;
};

message Contact {
    optional string phone_num = 1;
    optional string address = 2;
    optional ContactType ctype = 3; 
};

message Person {
    optional string name = 1;
    optional int32 id = 2;
    repeated int32 pid = 3;
    optional bytes contact = 4;
};
